ecc/scripts/esp-controller.js

```diff
@@ -1220,8 +1220,11 @@ export async function unpublishSeries(seriesId, seriesData) {
   if (!seriesId || typeof seriesId !== 'string') throw new Error('Invalid series ID');
   if (!seriesData || typeof seriesData !== 'object') throw new Error('Invalid series data');
 
+  const { targetCms, ...sanitizedData } = seriesData;
```

Comment:
You shouldn't need to do this. The filtering should be done in the data-utils, which I think you have done already.